app/controllers/PasskeyController.scala

```diff
@@ -143,33 +145,38 @@ class PasskeyController(
     )
   }
 
-  /** Deletes a passkey for a user */
-  def deletePasskey: Action[Map[String, Seq[String]]] = authAction(
-    parse.formUrlEncoded
-  ) { implicit request =>
-    apiResponse(
-      for {
-        passkeyId <- request.body.get("passkeyId") match {
-          case Some(values) => Success(values.head)
-          case None =>
-            Failure(
-              JanusException.missingFieldInRequest(request.user, "passkeyId")
+  /** Deletes a passkey from the user's account */
+  def deletePasskey(passkeyId: String): Action[AnyContent] = authAction {
```

Comment:
We're not expecting the request to have a body so:
```suggestion
  def deletePasskey(passkeyId: String): Action[Unit] = authAction(parse.empty) {
```

Reply:
Done in bb67182